.eslintrc.json

```diff
@@ -1,6 +1,12 @@
 {
```

Comment:
Your `.eslintrc` file does not seem to have the minimum rules that is required to properly format your code. You should consider getting the required configurations and update this file.